pandas/core/arrays/datetimes.py

```diff
@@ -820,7 +819,7 @@ def tz_convert(self, tz):
         dtype = tz_to_dtype(tz)
         return self._simple_new(self.asi8, dtype=dtype, freq=self.freq)
 
-    def tz_localize(self, tz, ambiguous="raise", nonexistent="raise"):
+    def tz_localize(self, tz, ambiguous="raise", nonexistent="raise"):  # noqa
```

Comment:
Why is the noqa needed here?

Reply:
another continuation line issue.

Reply:
Can we rather figure out what is wrong, or disable the indentation check?

Reply:
probably need a fix upstream

Reply:
looks like an issue has already been raised for this https://github.com/PyCQA/pydocstyle/issues/437